src/transformers/models/ctrl/modeling_tf_ctrl.py

```diff
@@ -702,7 +709,9 @@ def prepare_inputs_for_generation(self, inputs, past_key_values=None, use_cache=
         attention_mask = kwargs.get("attention_mask", None)
 
         if attention_mask is not None and position_ids is None:
-            position_ids = tf.math.cumsum(attention_mask, axis=-1, exclusive=True)
```

Comment:
this one should be correct, no? 🤔 

(the same comment applies to other TF models)